openmdao/core/total_jac.py

```diff
@@ -950,10 +950,7 @@ def simul_coloring_iter(self, imeta, mode):
 
         for color, ilist in enumerate(coloring.color_iter(mode)):
             if len(ilist) == 1:
-                if both:
```

Comment:
I think you can get rid of the variable `both` now.

Reply:
fixed